Automatic Validation ended with:
> 2024-04-23T11:10:31.0492744Z ##[error]Manifest Validation Failed
 2024-04-23T11:10:36.0998470Z ##[section]Finishing: Validate Manifest

(Automated response - build 877.)

--- a/manifests/b/BellSoft/LibericaJDK/21/Full/21.0.3.10/BellSoft.LibericaJDK.21.Full.installer.yaml
+++ b/manifests/b/BellSoft/LibericaJDK/21/Full/21.0.3.10/BellSoft.LibericaJDK.21.Full.installer.yaml
@@ -12,11 +12,11 @@ Installers:
   AppsAndFeaturesEntries:
   - DisplayName: Liberica JDK 21 Full (64-bit)
 - Architecture: x86
-InstallerUrl: https://download.bell-sw.com/java/21.0.3+10/bellsoft-jdk21.0.3+10-windows-i586-full.msi
-InstallerSha256: 19CACA2050ECC42472B1C0F44E48A63E98ECD77E5116CAC863D01E7E77DD0BDD
-ProductCode: '{E7CDCD5D-1AEA-4D7A-AF52-276F7BAA836A}'
-AppsAndFeaturesEntries:
-- DisplayName: Liberica JDK 21 Full (32-bit)
+  InstallerUrl: https://download.bell-sw.com/java/21.0.3+10/bellsoft-jdk21.0.3+10-windows-i586-full.msi
+  InstallerSha256: 19CACA2050ECC42472B1C0F44E48A63E98ECD77E5116CAC863D01E7E77DD0BDD
+  ProductCode: '{E7CDCD5D-1AEA-4D7A-AF52-276F7BAA836A}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Liberica JDK 21 Full (32-bit)
 - Architecture: arm64
   InstallerUrl: https://download.bell-sw.com/java/21.0.3+10/bellsoft-jdk21.0.3+10-windows-aarch64-full.msi
   InstallerSha256: 429E35453C51EC4A64E24B6713EDE3C4F90F5600B87E8808D861F6D821FD2A37

